### PR TITLE
Display patient sex and age in subtitle on PatientChartActivity

### PR DESCRIPTION
**Pull request by mjanes**
Solution to https://github.com/projectbuendia/client/issues/325

This does leave display of patient and sex and age in smaller font than previously, even if id and name is short. I am uncertain if that is acceptable, given visibility requirements.

A potential alternative would be using a custom view for action bar.

Prior view:

Screen Shot 2019-06-24 at 4 11 25 PM

New view, long name:

Screen Shot 2019-06-24 at 4 10 13 PM

New view, short name:

Screen Shot 2019-06-24 at 4 09 43 PM

### CODE REVIEW

**Comment by ivangayton** [comment]
Hi Matt,

This is a possible resolution to the problem in [#325](https://github.com/projectbuendia/client/issues/325), and it's lovely pull request; thanks for the great notes and screenshots!

 It's a 100% improvement in the case of overflow; the text may be small but it's present. It's no smaller than some of the other text on the screen, such as the dates on the chart, though it's maybe a bit harder to see due to lesser contrast.

I'm inclined to think it's an improvement over what we currently have, and is thus worth merging simply because if we never get around to fixing #325, small text is better than no text. Nothing stopping us from considering other solutions later if need be.

I'll leave it to Ping to make the final decision, but my vote is to merge this, close #325, and see if anyone complains that the age and sex are too small to see easily.

**Comment by mjanes** [comment]
Have adjusted style slightly. New appearance is easier to read.

Screen Shot 2019-06-25 at 11 33 56 AM

**Review by zestyping — APPROVED** [review]
Great, thanks!

**Pull request merged**
